<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(101, 252, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labeltext0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rbg(255, 255, 255);
border-radius: 15px; 
border-style: outset; 
border-width: 2px;</string>
    </property>
    <property name="lineWidth">
     <number>15</number>
    </property>
    <property name="midLineWidth">
     <number>15</number>
    </property>
    <property name="text">
     <string>WEATHER FORECAST</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
    <property name="margin">
     <number>-1</number>
    </property>
    <property name="indent">
     <number>7</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rbg(255, 255, 255);
border-radius: 15px; 
border-style: outset; 
border-width: 2px;</string>
    </property>
    <property name="placeholderText">
     <string>DIGITE SEU NOME</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rbg(255, 255, 255);
border-radius: 15px; 
border-style: outset; 
border-width: 2px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>DIGITE SUA SENHA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rbg(255, 255, 255);
border-radius: 15px; 
border-style: outset; 
border-width: 2px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>DIGITE SUA SENHA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(250, 216, 255);
border-radius: 15px; 
border-style: outset; 
border-width: 2px;
border-color: black;
}
QPushButton:pressed {
background-color: rgb(250, 216, 255);
border-style: outset; 
border-width: 2px;
border-radius: 15px; 
border-color: white;
}</string>
    </property>
    <property name="text">
     <string>Confirmar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>icon email.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>icon senha.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>icon senha.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="powerlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>POWERED BY: RAFAEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(250, 216, 255);
border-radius: 15px; 
border-style: outset; 
border-width: 2px;
border-color: black;
}
QPushButton:pressed {
background-color: rgb(250, 216, 255);
border-style: outset; 
border-width: 2px;
border-radius: 15px; 
border-color: white;
}</string>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
